streams/src/test/java/org/apache/kafka/streams/tests/StreamsNamedRepartitionTest.java

```diff
@@ -63,6 +63,7 @@ public static void main(final String[] args) throws Exception {
         final StreamsBuilder builder = new StreamsBuilder();
 
         final KStream<String, String> sourceStream = builder.stream(inputTopic, Consumed.with(Serdes.String(), Serdes.String()));
+        sourceStream.peek((k,v) -> System.out.println(String.format("input data key=%s, value=%s", k, v)));
```

Comment:
Used for debugging I'd prefer to leave in should the test have problems in the future